template/golang-http/Dockerfile

```diff
@@ -26,16 +26,16 @@ ENV CGO_ENABLED=${CGO_ENABLED}
 # Run a gofmt and exclude all vendored code.
 RUN test -z "$(gofmt -l $(find . -type f -name '*.go' -not -path "./vendor/*" -not -path "./function/vendor/*"))" || { echo "Run \"gofmt -s -w\" on your Golang code"; exit 1; }
 
-
 WORKDIR /go/src/handler/function
+RUN mkdir -p /go/src/handler/function/static
```

Comment:
we should update the README as well to document this flow

Reply:
Agreed, and the docs.